airbyte-integrations/connectors/source-paypal-transaction/source_paypal_transaction/schemas/transactions.json

```diff
@@ -5,135 +5,166 @@
           "properties": {
             "currency_code": {
-              "type": ["null", "string"]
+              "type": ["null", "string"],
+              "maxLength": 3
             },
             "value": {
-              "type": ["null", "string"]
+              "type": ["null", "number"]
             }
           }
         },
         "shipping_amount": {
           "type": ["null", "object"],
           "properties": {
             "currency_code": {
-              "type": ["null", "string"]
+              "type": ["null", "string"],
+              "maxLength": 3
             },
             "value": {
-              "type": ["null", "string"]
+              "type": ["null", "number"]
             }
           }
         },
         "shipping_discount_amount": {
           "type": ["null", "object"],
           "properties": {
             "currency_code": {
-              "type": ["null", "string"]
+              "type": ["null", "string"],
+              "maxLength": 3
             },
             "value": {
-              "type": ["null", "string"]
+              "type": ["null", "number"]
             }
           }
         },
         "transaction_status": {
-          "type": ["null", "string"]
+          "type": ["null", "string"],
+          "maxLength": 1
         },
         "transaction_subject": {
-          "type": ["null", "string"]
+          "type": ["null", "string"],
+          "maxLength": 256
         },
         "transaction_note": {
           "type": ["null", "string"]
         },
         "invoice_id": {
-          "type": ["null", "string"]
+          "type": ["null", "string"],
+          "maxLength": 127
         },
         "custom_field": {
           "type": ["null", "string"]
-        },
-        "protection_eligibility": {
-          "type": ["null", "string"]
         }
       }
     },
     "payer_info": {
       "type": ["null", "object"],
       "properties": {
         "account_id": {
-          "type": ["null", "string"]
+          "type": ["null", "string"],
+          "maxLength": 13
         },
         "email_address": {
-          "type": ["null", "string"]
+          "type": ["null", "string"],
+          "maxLength": 256
         },
         "address_status": {
-          "type": ["null", "string"]
+          "type": ["null", "string"],
+          "maxLength": 1
         },
         "payer_status": {
-          "type": ["null", "string"]
+          "type": ["null", "string"],
+          "maxLength": 1
         },
         "payer_name": {
           "type": ["null", "object"],
           "properties": {
             "given_name": {
-              "type": ["null", "string"]
+              "type": ["null", "string"],
+              "maxLength": 256
             },
             "surname": {
-              "type": ["null", "string"]
+              "type": ["null", "string"],
+              "maxLength": 256
             },
             "alternate_full_name": {
```

Comment:
```suggestion
```
Delete this property as it's been [deprecated](https://developer.paypal.com/docs/api/transaction-search/v1/#definition-name).